code/game/machinery/doors/firedoor.dm

```diff
@@ -74,6 +74,10 @@
 
 	check_atmos()
 
+	if(prob(0.1) && icon == 'icons/obj/doors/Doorfireglass.dmi')
```

Comment:
why is the capitalization here fucked?

Reply:
no idea, works all the same, if there were other changes I could fix it